files/config/cluster-config.json

```diff
@@ -1,11 +1,12 @@
 {
-  "electionMinTimeout": {{ $.Values.graphdb.clusterConfig.electionMinTimeout }},
-  "electionRangeTimeout": {{ $.Values.graphdb.clusterConfig.electionRangeTimeout }},
-  "heartbeatInterval": {{ $.Values.graphdb.clusterConfig.heartbeatInterval }},
-  "messageSize": {{ $.Values.graphdb.clusterConfig.messageSize }},
-  "verificationTimeout": {{ $.Values.graphdb.clusterConfig.verificationTimeout }},
+  "electionMinTimeout": {{ .Values.graphdb.clusterConfig.electionMinTimeout | int }},
+  "electionRangeTimeout": {{ .Values.graphdb.clusterConfig.electionRangeTimeout | int }},
+  "heartbeatInterval": {{ .Values.graphdb.clusterConfig.heartbeatInterval | int }},
+  "verificationTimeout": {{ .Values.graphdb.clusterConfig.verificationTimeout | int }},
+  "messageSizeKB": {{ .Values.graphdb.clusterConfig.messageSize | int }},
+  "transactionLogMaximumSizeGB": {{ .Values.graphdb.clusterConfig.transactionLogMaximumSizeGB | int }},
```

Comment:
this configuration is a decimal number, not sure how the int at the end will affect it

Reply:
True, I changed it to be passed as string cause the gdb API seems to mishandle real decimal numbers :shrug: